tinygrad/runtime/ops_metal.py

```diff
@@ -13,10 +13,17 @@ class _METAL:
   def __init__(self):
     self.mtl_buffers_in_flight: List[Any] = []
     self.device = Metal.MTLCreateSystemDefaultDevice()
+    self.dispatch_group = libdispatch.dispatch_group_create() 
     self.mtl_queue = self.device.newCommandQueue()
-  # TODO: is there a better way to do this?
+  def command_buffer(self):
+    command_buffer = self.mtl_queue.commandBuffer()
+    libdispatch.dispatch_group_enter(METAL.dispatch_group)
+    def leave(_): libdispatch.dispatch_group_leave(self.dispatch_group)
+    command_buffer.addCompletedHandler_(leave)
+    self.mtl_buffers_in_flight.append(command_buffer)
```

Comment:
is `self.mtl_buffers_in_flight` still needed ?

Reply:
You're right! It's obsolete now.

Reply:
Fixed